src/libstd/io/mod.rs

```diff
@@ -1332,10 +1332,11 @@ pub trait Seek {
 
     /// Returns the length of this stream (in bytes).
     ///
-    /// This method is implemented using three seek operations. If this method
-    /// returns successfully, the seek position is unchanged (i.e. the position
-    /// before calling this method is the same as afterwards). However, if this
-    /// method returns an error, the seek position is undefined.
+    /// This method is implemented using uo to three seek operations. If this
+    /// method returns successfully, the seek position is unchanged (i.e. the
+    /// position before calling this method is the same as afterwards).
+    /// However, if this method returns an error, the seek position is
+    /// undefined.
```

Comment:
I thought safe rust has no undefined behavoir? Is this UB anyway?

Reply:
Yes and no. The English meaning of "undefined behavior" is different from what most people mean by it. So what this function says is that the *position* is undefined. That's all, no other things can be undefined. People usually use UB to say "everything" is undefined -- including the contents of your `/root` directory and the state of your laundry.

I heard people use the term "unspecified behavior" to mean: "some things are not specified/undefined, but it's not the super bad UB".

I can change the word to "unspecified" if you think that is clearer?

Reply:
"the seek position is indetermined"?

Reply:
"unspecified" is used for similar situations elsewhere in the standard library, e.g. [It is unspecified how many elements are removed from the vector if the `Drain` value is leaked.](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.drain)

Reply:
I changed it to "unspecified" for now.